pysal/contrib/README.rst

```diff
@@ -52,8 +52,9 @@ Currently the following contribs are available:
 
     - .. versionadded:: 1.5
     - Path: pysal.contrib.viz
-    - Requires: `matplotlib`_, `pandas`_, `bokeh`_
+    - Requires: `matplotlib`_, `pandas`_
     - Optional:
+        * `bokeh` backend on `geoplot`: `bokeh`_
```

Comment:
Moved `bokeh` to optional too as it always imports on `try/except`, rather than directly (unlike `matplotlib` or `pandas`).